### ISSUE TITLE
Imac bug :"RyuSAK is not supported by this Mac" (13.1)

### ISSUE THREAD

**Issue by DarkOrigins74** [Ecks1337/RyuSAK#44]
**Describe the bug**
I correctly downloaded the app but whan i open it it says that my mac do not support it. Is it fixable ?

**To Reproduce**
1. Download the app
2. Click on the app
3. See error

**Expected behavior**
I expected the app to open

**Screenshots**

**Desktop (please complete the following information):**
- OS: Ventura 13.1
- Version 1.6

**Additional context**

**Comment by javire100** [comment]
Same problem here...

**Comment by furu00** [comment]
They missed to compile an x86 mac version. You are trying to run the ARM version.

**Comment by Swearie** [comment]
Same here...

**Comment by Kushagra3125** [comment]
Are there any binaries for intel macs?

**Comment by Swearie** [comment]
No, it's for arm cpus, that's why it cannot be installed on intel's. I
don't know why they didn't build one for intel cpus...

Il ven 12 mag 2023, 12:44 Kushagra3125 ***@***.***> ha
scritto:

**Comment by alessandro893** [comment]
> Are there any binaries for intel macs?

Mac intel x86 version - https://drive.google.com/file/d/1Nvm04muJ0qeOFy4_xcZqC188C8fL7NbB/view?usp=sharing
